# chassis_control_topic_name: /chassis/control_cmd
control_cmd_topic_name: /control_cmd

cansend_topic_name: /sent_messages

chassis_status_topic_name: /chassis/status

cansend_para:
  send_mode: 1 # 0 for test, 1 for autonomous driving 
  steer_angle: 0
  steer_scale: 16.5
  drive_target_speed: 3
  drive_target_gear: 1
  drive_limit_speed: 5

node_rate: 50   # [Herz]
